src/components/decimal/decimal.test.js

```diff
@@ -318,4 +333,107 @@ context("Tests for Decimal component", () => {
         });
     });
   });
+
+  describe("Accessibility tests for Card component", () => {
+    it("should pass accessibility tests for Decimal with different input sizes", () => {
+      CypressMountWithProviders(<Sizes />);
+
+      cy.checkAccessibility();
+    });
+
+    it("should pass accessibility tests for Decimal with label aligned left", () => {
+      CypressMountWithProviders(<LabelAlign />);
+
+      cy.checkAccessibility();
+    });
+
+    it("should pass accessibility tests for Decimal with custom precision", () => {
+      CypressMountWithProviders(<WithCustomPrecision />);
+
+      cy.checkAccessibility();
+    });
+
+    it("should pass accessibility tests for Decimal with custom label width and input width", () => {
+      CypressMountWithProviders(<WithCustomLabelWidthAndInputWidth />);
+
+      cy.checkAccessibility();
+    });
+
+    it("should pass accessibility tests for Decimal with custom max width", () => {
+      CypressMountWithProviders(<WithCustomMaxWidth />);
+
+      cy.checkAccessibility();
+    });
+
+    it("should pass accessibility tests for Decimal with field help", () => {
+      CypressMountWithProviders(<WithFieldHelp fieldHelp="Help" />);
+
+      cy.checkAccessibility();
+    });
+
+    it("should pass accessibility tests for Decimal with label help", () => {
+      CypressMountWithProviders(<WithLabelHelp labelHelp="Help" />);
+
+      getDataElementByValue("question")
+        .trigger("mouseover")
+        .then(() => {
+          // eslint-disable-next-line no-unused-expressions
+          cy.checkAccessibility();
+        });
+    });
+
+    it("should pass accessibility tests for Decimal required", () => {
+      CypressMountWithProviders(<Required />);
+
+      cy.checkAccessibility();
+    });
+
+    it("should pass accessibility tests for Decimal left aligned", () => {
+      CypressMountWithProviders(<LeftAligned />);
+
+      cy.checkAccessibility();
+    });
+
+    it("should pass accessibility tests for Decimal with validation", () => {
+      CypressMountWithProviders(<Validations />);
+
+      cy.checkAccessibility();
+    });
+
+    it("should pass accessibility tests for Decimal validations redesigned", () => {
+      CypressMountWithProviders(<ValidationsRedesign />);
+
+      cy.checkAccessibility();
+    });
+
+    // FE-5382
+    describe.skip("skip", () => {
+      it("should pass accessibility tests for Decimal with tooltip", () => {
+        CypressMountWithProviders(<ValidationsTooltip />);
+
+        cy.checkAccessibility();
+      });
+
+      it("should pass accessibility tests for Decimal with tooltip label", () => {
+        CypressMountWithProviders(<ValidationsTooltipLabel />);
+
+        cy.checkAccessibility();
+      });
+    });
+
+    // FE-4623
+    describe.skip("skip", () => {
```

Comment:
Why these tests are skipped? I can't see any of the issues on Carbon (with the same config for `axe`)?